charts/csi-baremetal-deployment/templates/csi-baremetal_v1_deployment.yaml

```diff
@@ -32,6 +32,12 @@ spec:
       log:
         format: {{ .Values.driver.log.format }}
         level: {{ .Values.driver.log.level }}
+      {{- if .Values.driver.controller.securityContext.enable }}
+      securityContext:
```

Comment:
Do we really need to extend API with these parameters? Can we just make them by default or modify docker files?

Reply:
In my opinion it's much more better to configure the behavior, which are initiated by external behavior (e.g. enabling cis hardening).
In a lot of examples at k8scluster-automation - other components uses the same approach.